src/parser/packet/message.rs

```diff
@@ -82,15 +83,14 @@ macro_rules! create_message_impl {
                     // The data from protobuf's `CSVCMsg_UserMessage` contains
                     // a `CUSRMsg_x`, so we need a new reader on those data to
                     // read our new message.
```

Comment:
Could you remove this comment please? `CodedInputStream` is not used anymore, and [the comment L77-L79](https://github.com/abenea/csgo-demo-parser/blob/6f9dff059eb2e3a7d179b46782a8ca8f344c7908/src/parser/packet/message.rs#L77-L79) already explains what this does.

Reply:
Done.